features/step_definitions/factory_steps.rb

```diff
@@ -17,6 +16,35 @@
   end
 end
 
+Given /^I accept a proposed organisation called "(.*?)" with email "(.*?)"$/ do |name, email|
+  steps %{
+    Given the following proposed organisations exist:
+      | name    | description         | address        | postcode | telephone | website            | email    | donation_info        | non_profit |
+      | #{name} | Mourning loved ones | 30 pinner road | HA5 4HZ  | 520800000 | http://#{name}.org | #{email} | www.pleasedonate.com | true       |
+    When I accept the proposed_organisation named "#{name}"
+  }
+end
+
+Then /^"(.*?)" is (notified|invited|ignored) as an organisation admin of "(.*?)"$/ do |email, action, name|
+  case action
+  when 'notified'
+    steps %{
```

Comment:
I guess it's a style preference. I use nested steps extensively, so maybe I'm just more accustomed to them. For instance, here is something I wrote yesterday:
```
When I submit the following banking information through the profile form:
  | Account Nickname    | Account Type | Routing Number | Account Number | Amount | Calculation Type |
  | My checking account | Checking     | 123456789      | 1234           | 88.88  | Percent          |
```
```ruby
When /^I submit the following banking information through the profile form:$/ do |table|
  hash = table.hashes.first
  steps %{
    And I enter "#{hash.fetch('Account Nickname')}" into a namely-ui input with a "Account Nickname" label
    And I select "#{hash.fetch('Account Type')}" from a namely-ui select labeled "Account Type"
    And I enter "#{hash.fetch('Routing Number')}" into a namely-ui input with a "Routing Number" label
    And I enter "#{hash.fetch('Account Number')}" into a namely-ui input with a "Account Number" label
    And I enter "#{hash.fetch('Amount')}" into a namely-ui input with a "100" placeholder
    And I select "#{hash.fetch('Calculation Type')}" from a namely-ui select labeled "Percent"
    And I click "Enroll in Direct Deposit"
  }
end
```
In this case, I'm using nested steps as a way to wrap a declarative cucumber table around some primitive page interaction steps. You're not a fan of this, though?

When it comes to debugging, I've noticed that my editor's macro for "jump to step definition" doesn't work in a nested step. So if I want to find its definition, I temporarily copy-paste it into some gherkin, and jump from there.